File: POL_Polowa/POL.cpp
#include <iostream>
#include <string>
using namespace std;

int main(){
    int t, l;
    string text;
    cin>>t;
    while(t--){
        cin>>text;
        l=text.length()/2;
        cout<<text.substr(0,l);
    }
    return(0);
}

File: POL_Polowa/POL.r
f <- file("stdin")
open(f, newline='')
t <- as.integer(readLines(f, n=1))

for (i in 1:t)
{
    text <- readLines(f, n=1)
    l=as.integer(nchar(text)/2)
    write(substr(text,1, l), stdout())
}
close(f)